### PR TITLE
Add a "Hello Microverse!" header

### PR DESCRIPTION
**Pull request by benjaMatteo96**
Hello, for this pull request:
- Add a readme file.
- Add .gitignore.
- Add an HTML file with a single h1 element.
- Add a CSS file with styles that apply to that h1 element.
-This pull request is using linters.

Thanks for watching this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+test_md
```
Comment (usorfaitheloho) [inline .gitignore:2]: You did well implementing the project requirements, weldone 👍 . kindly add the right file which in this case is `test.md` and not `test_md` to the `.gitignore` file, you did great adding the `node_modules` file, so this should be really easy 😄

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+h1 {
+  background-color: black;
+  color: white;
+  text-align: center;
+}
```
Comment (usorfaitheloho) [inline styles/style.css:5]: According to the project requirement, kindly add the text "This file should be ignored by git" in your `test.md` file below

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @benjaMatteo96  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjaMatteo96** [comment]
@usorfaitheloho Hi there!, thanks for the review, I really appreciate it.

**Review by usorfaitheloho — APPROVED** [review]
Hi @benjaMatteo96  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**